README.rst

```diff
@@ -409,8 +403,8 @@ Configuration files can be referenced anywhere in ``aws-encryption-cli`` paramet
 
    aws-encryption-cli -e -i $INPUT_DIR -o $OUTPUT_DIR @master-key.conf @caching.conf --recursive
 
-Configuration files can have many lines, include comments using ``#``, and include
-references to other configuration files.
+Configuration files can have many lines, include comments using ``#`` and escaped characters
```

Comment:
Minor: this reads a little awkwardly now. What about splitting it up into a couple sentences? Maybe something like: "Configuration files can have many lines, including comments using `#`. Escape characters are platform-specific (\ on Linux and MacOS, ` on Windows). Configuration files may also include references to other configuration files."